package.json

```diff
@@ -18,13 +18,13 @@
     "commander": "2.18.0",
     "console.table": "0.10.0",
     "download": "7.1.0",
-    "edgegrid": "3.0.5",
+    "edgegrid": "3.0.8",
```

Comment:
You might consider using the carrot notation here "^3.0.8" to indicate at least this version or newer.